views/_partials/header.jade

```diff
@@ -17,7 +17,7 @@
                 li: a(href="/bootlint/" + themeQs) Bootlint
                 li: a(href="/legacy/" + themeQs) Legacy
 
-            ul.nav.navbar-nav.navbar-right.visible-md.visible-lg
```

Comment:
I think these classes were designed to clean up the display on mobile. I would check this on a phone before merging to master.